README.md

```diff
@@ -1,41 +1,49 @@
-<img src="https://img.shields.io/badge/-CSESoc-blue" />
+[![CSEsoc](https://img.shields.io/badge/-CSESoc-blue)](https://www.csesoc.unsw.edu.au/)
+![Status](https://img.shields.io/website?label=status&up_message=online&url=https%3A%2F%2Fcircles.csesoc.unsw.edu.au%2F)
+[![Backend Tests](https://github.com/csesoc/Circles/actions/workflows/ci_BE.yml/badge.svg)](https://github.com/csesoc/Circles/actions/workflows/ci_BE.yml)
```

Comment:
Really minor nitpick, but maybe order it in fe -> be -> pylint? To make a clear grouping for be tests and pylint?

Reply:
easy fix; was made before merge